plugins/adr-common/api-report.md

```diff
@@ -0,0 +1,36 @@
+## API Report File for "@backstage/plugin-adr-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Entity } from '@backstage/catalog-model';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+import { ScmIntegrationRegistry } from '@backstage/integration';
+
+// @public
+export interface AdrDocument extends IndexableDocument {
+  date?: string;
+  status?: string;
+}
+
+// @public
+export type AdrFilePathFilterFn = (path: string) => boolean;
+
+// @public
+export const ANNOTATION_ADR_LOCATION = 'adr/location';
```

Comment:
```suggestion
export const ANNOTATION_ADR_LOCATION = 'backstage.io/adr/location';
```

Let's prepend backstage.io to make it easier to figure out where that this plugin is hosted externally.

Reply:
good catch 👍